tests/test_transforms_post_transforms.py

```diff
@@ -48,3 +48,12 @@ def missing_reference(app, env, node, contnode):
 
     content = (app.outdir / 'index.html').read_text(encoding='utf8')
     assert '<span class="n"><span class="pre">Age</span></span>' in content
+
+
+@pytest.mark.sphinx('html', testroot='transforms-post_transforms-keyboard',
+                    freshenv=True)
+def test_keyboard_issue_10495(app):
```

Comment:
I prefer to describe what problem this case confirms via function name.
```suggestion
def test_keyboard_splitting_keyword(app):
```

And it would be better to verify the result of the feature. In this case, the given key will be separated with a hyphen and represented as composite `<kbd>` tag:
```
assert '<kbd class="kbd docutils literal notranslate"><kbd class="kbd docutils literal notranslate">spanish</kbd>-<kbd class="kbd docutils literal notranslate">inquisition</kbd></kbd>' in (app.outdir / 'index.html').read_text(encoding='utf8')
```

Reply:
> And it would be better to verify the result of the feature.

All that matters here is that it doesn't crash -- this is a regression test so I'd say we don't fully care about the exact output.

Reply:
Renamed the test.